services/Booking.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Booking",

```

Comment:
As described in https://github.com/ambanum/CGUs/issues/106#issuecomment-681870747 (will soon be consolidated in the documentation), we preserve the service name when it includes a TLD, both in the name and the ID (file name) 🙂

```suggestion
    "name": "Booking.com",
```

The file name should also be updated.

![Capture d’écran 2020-10-15 à 15 15 38](https://user-images.githubusercontent.com/222463/96131413-6c352580-0ef9-11eb-82ff-2dbed70efef6.png)
![Capture d’écran 2020-10-15 à 15 15 57](https://user-images.githubusercontent.com/222463/96131418-6ccdbc00-0ef9-11eb-9057-1617975cb214.png)